### ISSUE TITLE
Error in bootstrap_icons version 1.10.5+1

### ISSUE THREAD

**Issue by liemfs** [ricardodalarme/Bootstrap-Icons-Flutter#2]
Target of URI doesn't exist: 'package:bootstrap_icons/bootstrap_icons.dart'.
Try creating the file referenced by the URI, or try using a URI for a file that does exist.

Missing lib folder in package

**Comment by erhacorpdotcom2022** [comment]
I have the error too
Missing lib folder in package

**Comment by liemfs** [comment]
> I have the error too Missing lib folder in package

Quick fix, change your pubspec.yaml
```
bootstrap_icons: 
    git:
      url: https://github.com/ricardodalarme/Bootstrap-Icons-Flutter
```

**Comment by ricardodalarme** [comment]
Fixed with 1.10.5+2, thank you!!
